### PR TITLE
app-misc/elasticsearch: bump to 5.3.2

### PR DESCRIPTION
**Pull request by hydrapolic**
Package-Manager: Portage-2.3.5, Repoman-2.3.2

https://bugs.gentoo.org/show_bug.cgi?id=616556

### CODE REVIEW

File: app-misc/elasticsearch/elasticsearch-5.3.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+inherit eutils systemd user
+
+DESCRIPTION="Open Source, Distributed, RESTful, Search Engine"
+HOMEPAGE="https://www.elastic.co/products/elasticsearch"
+SRC_URI="https://artifacts.elastic.co/downloads/${PN}/${P}.tar.gz"
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="strip"
```
Comment (mgorny) [inline app-misc/elasticsearch/elasticsearch-5.3.2.ebuild]: Is there anything strippable in the package?

File: app-misc/elasticsearch/elasticsearch-5.3.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils systemd user
```
Comment (mgorny) [inline app-misc/elasticsearch/elasticsearch-5.3.2.ebuild]: Is eutils being actually used anywhere?

File: app-misc/elasticsearch/elasticsearch-5.3.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+DESCRIPTION="Open Source, Distributed, RESTful, Search Engine"
+HOMEPAGE="https://www.elastic.co/products/elasticsearch"
+SRC_URI="https://artifacts.elastic.co/downloads/${PN}/${P}.tar.gz"
+LICENSE="Apache-2.0"
```
Comment (mgorny) [inline app-misc/elasticsearch/elasticsearch-5.3.2.ebuild]: I suspect there might be a similar issue to kibana, with those .jars.

**Comment by mgorny** [comment]
@gentoo/java, a second look would be appreciated.

**Comment by hydrapolic** [comment]
@mgorny, yes you are right, eutils was probably a remnant from the paste and there is nothing strippable in the package. The licenses are a bit tricky, I tried to collect as much as I could.

**Comment by hydrapolic** [comment]
Thanks for the tips and for merging @mgorny.

**Pull request closed without merging**